### ISSUE TITLE
Question mark help icons are not tab navigable

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1157]
question mark help icons on edit dialogs are not tab navigable

**Comment by sudo-may** [comment]
after further investigation help icons should not be tab navigable since they are not actionable; rather buttons/links should have proper aria-label description the action
